util/src/builder/callback_data.rs

```diff
@@ -0,0 +1,175 @@
+use twilight_model::{
+    application::{callback::CallbackData, component::Component},
+    channel::{
+        embed::Embed,
+        message::{AllowedMentions, MessageFlags},
+    },
+};
+
+/// Builder for [`CallbackData`].
+///
+/// # Example
+/// ```
+/// use twilight_util::builder::CallbackDataBuilder;
+/// use twilight_model::channel::message::MessageFlags;
+///
+/// let callback_data = CallbackDataBuilder::new()
+///     .content("My callback message")
+///     .flags(MessageFlags::EPHEMERAL)
+///     .build();
+/// ```
+#[derive(Debug)]
+#[must_use = "builders have no effect if unused"]
+pub struct CallbackDataBuilder(CallbackData);
+
+impl CallbackDataBuilder {
+    /// Create a new builder to construct a [`CallbackData`].
+    pub fn new() -> Self {
+        Self::default()
+    }
+
+    /// Set allowed mentions in the interaction response.
+    #[allow(clippy::missing_const_for_fn)]
```

Comment:
is this necessary?

Reply:
Having a `const fn` produce the same compilation error as the `content` method, but this trigger a Clippy lint.